README.md

```diff
@@ -25,25 +25,25 @@ All automated.
     - [Setup a VPN Container](#setup-a-vpn-container)
       - [Introduction](#introduction)
       - [privateinternetaccess.com custom setup](#privateinternetaccesscom-custom-setup)
-      - [Docker container](#docker-container)
+      - [Docker container](#docker-container-1)
```

Comment:
These could be easier to maintain with more explicit names. For example: "#docker-container-vpn", "#docker-container-radarr", etc.

Reply:
I thought the same thing originally, but I felt that I should submit a fix without changing the content.  I could update it so they have unique names if you'd like!

Side note, when I wrote the section on Bazarr, I did it your way specifically for that reason haha